docs/docs/03-main-concepts.md

```diff
@@ -0,0 +1,79 @@
+## Main Concepts
+
+
+### Unbundled Development
+
+![webpack vs. snowpack diagram](/img/snowpack-unbundled-example-3.png)
+
+**Unbundled development** is the idea of shipping individual files to the browser during development. Files can still be built with your favorite tools (like Babel, TypeScript, Sass) and then loaded individually in the browser with dependencies thanks to ESM `import` and `export` syntax. Any time you change a file, Snowpack only ever needs to rebuild that single file.
```

Comment:
I like highlighting and emphasizing this “unbundled” term. I think it communicates a lot!